src/bootstrap/lib.rs

```diff
@@ -138,18 +138,10 @@ const EXTRA_CHECK_CFGS: &[(Option<Mode>, &str, Option<&[&'static str]>)] = &[
     (Some(Mode::Std), "freebsd13", None),
     (Some(Mode::Std), "backtrace_in_libstd", None),
     /* Extra values not defined in the built-in targets yet, but used in std */
-    // #[cfg(bootstrap)]
-    (Some(Mode::Std), "target_vendor", Some(&["unikraft"])),
     (Some(Mode::Std), "target_env", Some(&["libnx"])),
-    // #[cfg(bootstrap)] hurd
-    (Some(Mode::Std), "target_os", Some(&["teeos", "hurd"])),
+    (Some(Mode::Std), "target_os", Some(&["teeos"])),
```

Comment:
Was added in 1.73 by https://github.com/rust-lang/rust/commit/72dd53c8e52e3f6995cfa29a66dcde28a691beb8
```suggestion
    // (Some(Mode::Std), "target_os", Some(&[])),
```